package.json

```diff
@@ -2,13 +2,21 @@
   "name": "dashboard",
   "version": "0.0.1",
   "dependencies": {
+    "@apollo/react-hooks": "^3.1.1",
+    "@fortawesome/fontawesome-free": "5.11.2",
+    "@fortawesome/react-fontawesome": "0.1.8",
+    "apollo-boost": "^0.4.4",
     "bootstrap": "~4.3.1",
-    "d3": "^5.9.4",
+    "d3": "^5.12.0",
     "d3-tip": "~0.9.1",
+    "graphql": "^14.5.7",
     "jquery": ">=3.4.0",
-    "rc-slider": "^8.6.13",
-    "@fortawesome/fontawesome-free": "5.11.2",
-    "@fortawesome/react-fontawesome": "0.1.8"
+    "lodash.debounce": "^4.0.8",
+    "lodash.isequal": "^4.5.0",
+    "rc-slider": "^8.7.1",
+    "react-dnd": "^9.3.4",
+    "react-dnd-html5-backend": "^9.3.4",
```

Comment:
@justin0022, you did not end up using these, correct? I think we might want to remove them for now (and commented out lines elsewhere, which I think I saw). We can make this a separate issue to explore further in the future.